extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+##### Complete Levels
+
+For tilesets that have uniform subdivision for each tile up to a certain level, it is redundant to store the tile subdivision state information. The `completeLevels` property is used to indicate how many levels within the tileset are complete. Complete levels are levels in which all tiles subdivide implicitly and internally (using the bitcode `11`).
+
+#### Content
+
+This is a one bit representation of whether or not a tile has content associated with it.
+
+| Code | Description       |
+|------|-------------------|
+| `0`  | No content        |
+| `1`  | Has content       |
+
+When the tile has content, the content payload can be found implicitly by combining the tile location with the tile extension.
+
+#### Metadata
+
+This is a one bit representation of whether or not a tile has content associated with it.
+
+| Code | Description       |
+|------|-------------------|
+| `0`  | No metadata       |
+| `1`  | Has metadata      |
+
+
+When the tile has content, the index of the tile in the subdivision buffer will be the index of corresponding metadata property in the associated buffer in the `3DTILES_tile_metadata` extension.
+
+#### Level Offsets
+
+In the content and metadata objects, a level offset and a default value can be specified to implicitly apply the default value to each tile in all levels before the specified level offset. Similar to 
+
+#### Level Offset Fill
+
+The level offset fill describes the state of all tiles in levels before the level offset.
+
+### Bitstream Layout
+
+Each bitstream encodes tile state information using an adaptive linear quadtree or octree (depending on the `tilingScheme`) representation of the hierarchy. Tile states are packed in increasing order of level and Morton index of tiles within that level. Only tiles that subdivide internally will have information about their children present at the next level. This allows for a space efficient representation of sparse hierarchies.
+
+*The following section is non-normative.*
+
+The following example illustrates the usage of these buffers in a sparse quadtree that extends to 2 levels. 
+
+```json
+{
+    "tilingScheme": "quadtree",
+    "subdivision": {
+        "bufferView": 0
+    },
+    "content": {
+        "levelOffset": 2,
+        "levelOffsetFill": 1,
+        "bufferView": 1
+    },
+    "metadata": {
+        "levelOffset": 2,
+        "levelOffsetFill": 0,
+        "bufferView": 0
+    }
+}
+```
+
+Only the information with grey background is present in the bitstream.
+
+![States](figures/states.png)
+
+### Root Tiles
+
+Multiple tilesets using this extension can be combined by using parent tileset referring to one or more external tilesets using `3DTILES_implicit_tiling` as the children of the root tile.
+
+### Jump Buffer
+
+The jump buffer is a buffer created at runtime to allow efficient traversal and tile lookup functions. The jump buffer is a bijection between a tile's Morton index and its index in the level.
```

Comment:
Anything jump buffer related should be non-normative.

Reply:
Moved jump buffer to Appendix.